.gitignore

```diff
@@ -25,3 +25,5 @@ models/rover/rover.sdf
 models/tiltrotor/tiltrotor.sdf
 models/boat/boat.sdf
 models/typhoon_h480/typhoon_h480.sdf
+models/temp*
+worlds/temp*
```

Comment:
I can't find my previous review anymore, what is stopping us from moving this to `/tmp`?